exercises/practice/ocr-numbers/.docs/instructions.md

```diff
@@ -27,7 +27,7 @@ Is converted to "0"
 
 Is converted to "1"
 
-If the input is the correct size, but not recognizable, your program should return '?'
+If the input is the correct size, but not recognizable, your program should return "?"
```

Comment:
All the instructions documents come from a language-independent central repository (problem-specifications) and we cannot make Elixir-specific changes in them. They would be overwritten the next time we synchronize with that repository anyway:
```suggestion
If the input is the correct size, but not recognizable, your program should return '?'
```